### PR TITLE
updating the doc for more clarity

### PR DESCRIPTION
**Pull request by thesteve0**
This is an update to make the doc more modular so it flows better for people building containers. I also added clarifying statement into pull requests. Please review for accuracy, grammar, and general goodness

### CODE REVIEW

File: hugo/content/contributing/pull-requests.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ weight: 253
 
 So you decided to submit an issue and work it.  Great! Let's get it merged in to the codebase. The following will go a long way to helping get the fix merged in quicker:
 
-1. Create a Pull Request from your Fork to the Develop branch.
-2. Update the checklists in the Pull Request description.
-3. Reference which issues this Pull Request is resolving.
+1. Fork the [Github repository](https://github.com/CrunchyData/crunchy-containers) and make a branch off of the *develop* branch
+2. Create a Pull Request from your Fork back to the *develop* branch.
```
Comment (davecramer) [inline hugo/content/contributing/pull-requests.md:11]: I think you can link to the branch using *[develop]*(/CrunchyData/crunchy-containers/tree/develop)

Comment (thesteve0) [reply hugo/content/contributing/pull-requests.md:11]: Thanks! Done

**Comment by thesteve0** [comment]
@cahoonpwork @jmccormick2001 any chance to merge this?

**Review by cahoonpwork — APPROVED** [review]

**Comment by cahoonpwork** [comment]
@thesteve0  sorry looks I forgot to mark the review as complete.  @jmccormick2001  this one should be good now.

**Pull request merged**